resolwe_bio/descriptors/general_clinical.yml

```diff
@@ -0,0 +1,1135 @@
+          required: false
+        - name: annotator
+          label: Annotator
+          type: basic:string
+          required: false
+    - name: subject_information
+      label: Subject information
+      group:
+        - name: sample
+          label: Sample
+          type: basic:string
+          required: false
+        - name: subject_id
+          label: Subject ID
+          type: basic:string
+          required: false
+        - name: batch
+          label: Batch
+          type: basic:integer
+          required: false
+        - name: group
+          label: Group
+          type: basic:integer
+          required: false
+        - name: gender
+          label: Gender
+          type: basic:string
+          required: false
+          choices:
+            - label: M
+              value: m
+            - label: F
+              value: f
+        - name: race
+          label: Race
+          type: basic:string
+          required: false
+          choices:
+            - label: White
+              value: white
+            - label: Non-White
+              value: non-white
+            - label: Asian
+              value: asian
+            - label: Non-Asian
+              value: non-asian
+            - label: Black
+              value: black
+            - label: Non-Black
+              value: non-black
+        - name: ethnicity
+          label: Ethnicity
+          type: basic:string
+          required: false
+          choices:
+            - label: African Americans
+              value: african americans
+            - label: Non-African Americans
+              value: non-african americans
+            - label: Hispanic
+              value: hispanic
+            - label: Non-Hispanic
+              value: non-hispanic
+            - label: American Indians and Alaska Natives
+              value: american indians and alaska natives
+            - label: Asians and Pacific Islanders
+              value: asians and pacific islanders
+            - label: Non-Asians and Pacific Islanders
+              value: non-asians and pacific islanders
+        - name: age_in_years
+          label: Age in years
+          type: basic:decimal
+          required: false
+        - name: height
+          label: Height
+          type: basic:decimal
+          required: false
+        - name: weight
+          label: Weight
+          type: basic:decimal
+          required: false
+        - name: archival
+          label: Archival
+          type: basic:string
+          required: false
+          choices:
+            - label: "Yes"
+              value: "yes"
+            - label: "No"
+              value: "no"
+        - name: subject_status
+          label: Subject status
+          type: basic:string
+          required: false
+          choices:
+            - label: Alive
+              value: alive
+            - label: Dead
+              value: dead
+            - label: Unknown
```

Comment:
This may be redundant. If we don't have the information, we can leave this field empty because it's not required. Similar to `ethnicity`.